pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "syncstorage-rs"
+version = "0.1.0"
+description = "sync storage server in rust"
+authors = ["Taddes <tkorris@mozilla.com>"]
```

Comment:
Kinda wondering if we should switch to the general delivery address for this? `"Mozilla Services Engineering <services-engineering+code@mozilla.com>"`
If nothing else, I should make sure that the delivery list for that email address is up-to-date.

Reply:
Yeah, this was just the autogen when I did the `poetry init`, was going to change it to a group delivery option as I got the details in place.

Reply:
Heh, no worries. Honestly? Thank you. I realized that we had accidentally pruned the old group email, so I rebuilt it.